pkg/loki/modules.go

```diff
@@ -306,6 +314,8 @@ func (t *Loki) initQueryFrontend() (_ services.Service, err error) {
 	t.server.HTTP.Handle("/api/prom/label", frontendHandler)
 	t.server.HTTP.Handle("/api/prom/label/{name}/values", frontendHandler)
 	t.server.HTTP.Handle("/api/prom/series", frontendHandler)
+	t.server.HTTP.Handle("/loki/api/v1/tail", wsHandler)
+	t.server.HTTP.Handle("/api/prom/tail", wsHandler)
 	// fallback route
 	t.server.HTTP.PathPrefix("/").Handler(frontendHandler)
```

Comment:
If we are going with reverse proxy as I mentioned above, I think we should set that handler here which should automatically take care of special cases like WebSocket. @cyriltovena @slim-bean what do you guys think?

Reply:
I will consider your proposal and we will see.